src/Backend/Modules/MediaLibrary/Domain/MediaGroup/SingleMediaGroupType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary\Domain\MediaGroup;
+
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SingleMediaGroupType extends MediaGroupType
```

Comment:
Can you use final class instead?

Reply:
Shouldn't people be able to extend this?

Reply:
In this case I see no purpose in *not* adding final as there is barely anything in this class. If you want to protect the "single" aspect of this class but allow the fields to be added or edited, you should use `final public function configureOptions(OptionsResolver $resolver): void` instead.

Reply:
fixed in 84965db65